package.json

```diff
@@ -207,15 +207,15 @@
     "scripts": {
         "vscode:prepublish": "tsc -p ./",
         "compile": "tsc -watch -p ./",
-        "lint": "tslint --project tsconfig.json -e src/*.d.ts --type-check -t verbose",
+        "lint": "tslint --project tsconfig.json -e src/*.d.ts -t verbose",
```

Comment:
why this change?

Reply:
I was getting this message when I ran the lint task:
```
--type-check is deprecated. You only need --project to enable rule which need type information.
```